cmd/alerts/utils/utils.go

```diff
@@ -15,14 +16,35 @@ const (
 	AccountNamespace = "openshift-monitoring"
 	ContainerName    = "alertmanager"
 	LocalHostUrl     = "http://localhost:9093"
-	PodName          = "alertmanager-main-0"
+	PrimaryPod       = "alertmanager-main-0"
+	SecondaryPod     = "alertmanager-main-1"
 )
 
+// ExecInPod is designed to execute a command inside a Kubernetes pod and capture its output.
 func ExecInPod(kubeconfig *rest.Config, clientset *kubernetes.Clientset, cmd []string) (string, error) {
+	var cmdOutput string
+	var err error
 
-	req := clientset.CoreV1().RESTClient().Post().Resource("pods").Name(PodName).
-		Namespace(AccountNamespace).SubResource("exec")
+	// Attempt to execute with the primary pod
+	cmdOutput, err = ExecWithPod(kubeconfig, clientset, PrimaryPod, cmd)
+	if err == nil {
+		return cmdOutput, nil // Successfully executed
+	}
+
```

Comment:
We should log the above error here before informing the user that we're attempting to fallback to the secondary pod.

Reply:
```
cmdOutput, err = ExecWithPod(kubeconfig, clientset, PrimaryPod, cmd)
	if err == nil {
		return cmdOutput, nil
	}

	fmt.Printf("Execution with alertmanager-main-0 failed: %v\n", err)

	fmt.Println("Attempting with alertmanger-main-1")
	cmdOutput, err = ExecWithPod(kubeconfig, clientset, SecondaryPod, cmd)
	if err == nil {
		return cmdOutput, nil
	}

	fmt.Printf("Execution with alertmanager-main-1 failed: %v\n", err)
```

Reply:
Current execution returns 
```

Execution with alertmanager-main-0 failed: failed to stream with context: unable to upgrade connection: container not found ("alertmanager")
Attempting with alertmanger-main-1
Alert Information:
  AlertName:  AlertmanagerSilencesActiveSRE
  Severity:   warning
  State:      active
  Message:
```